webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -139,6 +141,12 @@ function SystemUsers(props: Props) {
         props.tablePropertyDateRange,
     ]);
 
+    function handleDismissMySqlNotice() {
+        setShowMySqlBanner(false);
+
+        // TODO more?
```

Comment:
Waiting on a response as to if we want a user pref or some other storage item to make sure this isn't shown again.